### ISSUE TITLE
Fonts in validation UI of gene selector are needlessly large 

### ISSUE THREAD

**Issue by alisman** [cBioPortal/cbioportal#10736]
![image](https://github.com/cBioPortal/cbioportal/assets/186521/672b1e55-f19b-4bfd-9ac5-e4d52cb43c7a)

**Comment by Nithin9585** [comment]
I agree that the font size in the validation UI of the gene selector might be a bit large. It could be improved for better readability and space efficiency.  I have experience with CSS and might be able to help adjust the font size if needed. Are there specific areas of the validation UI that seem most problematic?

**Comment by mhsh312** [comment]
Hi there. I understand that assigning the problematic text the same css class as the text below it would fix the issue. 
![image](https://github.com/cBioPortal/cbioportal/assets/135870090/1d9bece9-574f-4c99-99f7-86d4887ec2bc)
Something similar can be arranged for the other example as well. 
So, please assign this issue to me.

**Comment by LakshayPuhal** [comment]
@alisman I would like to work on this issue.

**Comment by KhushamBansal** [comment]
Hi @dippindots , I would love to work on this issue. Could you please assign this to me?

**Comment by Tejas-Nahargadkar** [comment]
Hello,
I’m  interested in contributing to cBioPortal and would love to work on this issue. The font sizes in the validation UI of the gene selector seem larger than necessary, and I’d like to help improve the user experience by adjusting them to a more appropriate size.

I am currently exploring cBioPortal as part of my application for Google Summer of Code (GSoC) and believe this issue would be a great way to start contributing. Could you please assign this issue to me?

Looking forward to your guidance!

Thanks!

**Comment by dippindots** [comment]
@KhushamBansal @Tejas-Nahargadkar Please feel free to build frontend project on your local and try to make a pull request. We don't assign tickets but will review any ready-to-review pull requests. Thank you!

**Comment by ArshOnGit1607** [comment]
i would like to start my contribution for cbioportal , can i be assigned for this

**Comment by dippindots** [comment]
@ArshOnGit1607 Please see the message above. As far as I know, no one is currently working on this ticket, so you’re welcome to proceed and open a pull request.

**Comment by rujuthaa** [comment]
@alisman @dippindots  I opened a PR for this issue: https://github.com/cBioPortal/cbioportal-frontend/pull/5417

**Comment by alisman** [comment]
seems fixed
